routes/confirmation/confirmation.controller.js

```diff
@@ -1,10 +1,19 @@
 const { checkSchema } = require('express-validator')
-const { doRedirect, renderWithData, checkErrors } = require('./../../utils')
+const { doRedirect, renderWithData, checkErrors, getPreviousRoute } = require('./../../utils')
 const { reviewSchema } = require('./../../schemas')
+const { formatAnswerInfo } = require('./checkAnswersFormat')
 
 module.exports = function(app) {
   app.get('/confirmation', renderWithData('confirmation/confirmation'))
 
   app.get('/review', renderWithData('confirmation/review'))
   app.post('/review', checkSchema(reviewSchema), checkErrors('confirmation/review'), doRedirect)
+
+  app.get('/checkAnswers', (req, res) => {
+    res.render('confirmation/check-answers', {
+      data: req.session,
+      prevRoute: getPreviousRoute(req),
+      answerInfo: formatAnswerInfo(req.session),
+    })
+  })
```

Comment:
Maybe we could make `renderWithData` take an `options` object and then you could put them into the template.